otp-client-library/src/commonMain/kotlin/edu/usf/cutr/otp/plan/model/RequestParameters.kt

```diff
@@ -37,6 +37,7 @@ data class RequestParameters
     val triangleSafetyFactor: Double? = null,
     val wheelchair: String? = null,
     val date: String? = null,
+    val time: String? = null,
     val numItineraries: Int? = 3,
```

Comment:
After we've clarified the supported date and time formats, could you add them here in comments in another PR?